rrfs-test/validated_yamls/templates/obtype_config/msonet_airTemperature_188.yaml

```diff
@@ -83,61 +95,98 @@
              is_in: 188
            action:
              name: inflate error
-             #inflation factor: 1.09757
              inflation variable:
                name: ObsFunction/ObsErrorFactorPressureCheck
                options:
                  variable: airTemperature
-                 inflation factor: 0.0
-                 #geovar_sfc_geomz: surface_geometric_height #FV3
-                 #geovar_sfc_geomz: surface_altitude #MPAS
-           defer to post: true
+                 inflation factor: 8.0
 
+         # Bounds Check
          - filter: Bounds Check
            apply at iterations: 0,1
            filter variables:
            - name: airTemperature
            minvalue: 100
            maxvalue: 400
 
+         # Create temporary ObsErrorData
+         - filter: Variable Assignment
+           assignments:
+           - name: TempObsErrorData/airTemperature
+             type: float
+             function:
+               name: ObsFunction/Arithmetic
+               options:
+                 variables:
+                 - name: ObsErrorData/airTemperature
+
+         # Set ObsError set "error parameter" if < "max value"
+         - filter: Perform Action
+           filter variables:
+           - name: airTemperature
+           action:
+             name: assign error
+             error parameter: &error_min 1.0
```

Comment:
The same concern applies to the error_min and error_max for T, Q, and PSFC observations.